src/DataObjects/WorkflowDefinition.php

```diff
@@ -430,7 +430,7 @@ public function getDefaultWorkflowTitle()
         $tmp = [];
 
         foreach ($defs as $def) {
-            $parts = preg_split("#\s#", $def, -1, PREG_SPLIT_NO_EMPTY);
+            $parts = preg_split("#\s#", preg_quote($def), -1, PREG_SPLIT_NO_EMPTY);
```

Comment:
```suggestion
            $parts = preg_split("#\s#", preg_quote($def, '#'), -1, PREG_SPLIT_NO_EMPTY);
```
I'm not certain that the quote needs to pass the delimiter, but it can't hurt